### ISSUE TITLE
Consider removing <abbr> as an example in Body component

### ISSUE THREAD

**Issue by sarahpulis** [govau/design-system-components#596]
# Bug Report

- [x] I’ve read and understood the [Contributing guidelines](https://github.com/govau/design-system-components/blob/master/CONTRIBUTING.md) and have done my best effort to follow them.
- [x] I’ve read and agree to the [Code of Conduct](https://github.com/govau/design-system-components/blob/master/CODE_OF_CONDUCT.md).
- [x] I’ve searched for any related issues and avoided creating a duplicate issue.

First I want to acknowledge that the <abbr> element is a valid HTML element and is even recommended for use in WCAG Technique [H28: Providing definitions for abbreviations by using the abbr element](https://www.w3.org/TR/WCAG20-TECHS/H28.html).

However, the <abbr> element is not accessible to keyboard users or voice recognition users (e.g. Dragon NaturallySpeaking). These users cannot "hover" over the abbreviation or acronym meaning they cannot see the expanded version.

I recommend that the <abbr> element is removed from the body components, otherwise it looks like <abbr> is a recommended technique for acronyms. An alternative technique is using plain old standard text content for acronyms (e.g. WCAG Technique [G97: Providing the first use of an abbreviation immediately before or after the expanded form](https://www.w3.org/TR/WCAG20-TECHS/G97.html) which is much more robust.

**Comment by alex-page** [comment]
Hi Sarah thanks for the issue, I agree with what you are stating here.

The recommendation from WCAG does say use the text content before abbreviation. Do you think the abbreviation tag could be useful after they have described the abbreviation? e.g:

> I work at the Digital Transformation Agency ( **DTA** ). The **DTA** is a lot of fun.

The bolded DTA's could use the abbreviation tag?

I think we should definitely update our documentation to reflect what you are describing as we do not want people to think this is a solution for abbreviations.

**Comment by sarahpulis** [comment]
Hey Alex. In the example you've given, a screen reader users would hear the expanded version twice.

So, for Digital Transformation Agency (DTA) where DTA uses the <abbr> element, they'd hear something like "Digital Transformation Agency, DTA, Digital Transformation Agency.

The same would occur for the second DTA - my opinion is the visual experience should match the screen reader experience, so it is unnecessary to use the <abbr> element for the second DTA.

**Comment by alex-page** [comment]
Ok lets agree to remove it then 👍, and add some documentation around abbreviations.

**Comment by sukhrajghuman** [comment]
Hey @sarahpulis, thanks for bringing this up. I just have one question.

So if we did an abbreviation like 

```html
<p> Pounds per square inch (PSI). The recommended is 35 PSI </p>
```

The screen reader (voiceover and safari) reads

> "Pounds per square inch si. The recommended is 35 si. "

We could use contextual abbreviation for the first mention, and then use `<abbr>PSI</abbr>` for any mentions following the first. It would be good for those users who use screen readers that _do_ support the `<abbr>` tag, and no difference for those who use screen readers that don't.

The other solution would be just to put full stops between each letter in an acronym, but that may conflict with the [content guide](https://guides.service.gov.au/content-guide/terms-phrases/#acronyms).

What are your thoughts?

**Comment by alex-page** [comment]
I agree with @sarahpulis I don't think we should be using the `<abbr>`. 

I think this is worth a discussion in the future around if we should use `P.S.I.` or `PSI`. My opinion is that screen readers should read out `PSI` correctly. If they don't that seems like an oversight as it is the common practice and recommended by many content guides ( UK ).

I don't see any issues removing the examples so users don't get confused with lots of acronyms being read out.

**Comment by alex-page** [comment]
@sarahpulis We have removed `<abbr>` and added some rationale. Happy to reopen this if you have any feedback.
